runtime/envelope/envelope.go

```diff
@@ -412,10 +347,6 @@ func (e *Envelope) copyLines(component string, src io.Reader) error {
 	}
 }
 
-func (e *Envelope) Weavelet() *protos.WeaveletInfo {
-	return e.weavelet
-}
```

Comment:
This change will require a patch to weaver-gke, which I've prepared in a branch: https://github.com/ServiceWeaver/weaver-gke/compare/main...dnephin:weaver-gke:remove-envelope-weavlet-method

I can open a PR for that change if this change is accepted.

Reply:
Ah, maybe not! https://github.com/ServiceWeaver/weaver-gke/pull/31 removes this call already.